web/scripts/assets/FontManifest.json

```diff
@@ -0,0 +1 @@
+[]
```

Comment:
I understand why this is here, but on the face of it it's a bit mysterious. Can we doc why this is here somewhere?

Reply:
required. can discuss offline

Reply:
Yes, I understand the reason, but it would be useful to doc somewhere why this empty file is here.